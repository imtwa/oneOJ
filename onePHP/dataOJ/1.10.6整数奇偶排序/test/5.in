55 97 30 38 51 21 40 22 89 69 