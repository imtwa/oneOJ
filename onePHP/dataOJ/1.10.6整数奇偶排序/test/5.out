97 89 69 55 51 21 22 30 38 40 